build.psm1

```diff
@@ -534,7 +554,7 @@ function Get-InstalledCLIVersion {
         # earlier versions of dotnet do not support --list-sdks, so we'll check the output
         # and use dotnet --version as a fallback
         $sdkList = & $script:DotnetExe --list-sdks 2>&1
-        $sdkList = "Unknown option"
+        # $sdkList = "Unknown option"
```

Comment:
In PR #1367 you will remove this line anyway, so I suggest to revert the change in this line or just delete the whole line to avoid a merge conflict

Reply:
done!